commcare-cloud/commcare_cloud/commands/inventory_lookup/inventory_lookup.py

```diff
@@ -92,9 +96,16 @@ def run(self, args, manage_args):
         # the paths here are redundant with ansible/group_vars/all.yml
         code_current = '/home/{cchq_user}/www/{deploy_env}/current'.format(
             cchq_user=cchq_user, deploy_env=deploy_env)
-        remote_command = (
+        remote_command = [
```

Comment:
I think I'd no longer call this variable `remote_command`, since it may now be `-t` followed by the remote command

Reply:
5ddbb8a99636a70c2180dc39908fba2e115c29fb